addons/project/models/project_task.py

```diff
@@ -602,7 +602,7 @@ def _compute_subtask_count(self):
             )
         }
         for task in self:
-            task.subtask_count, task.closed_subtask_count = total_and_closed_subtask_count_per_parent_id.get(task.id, (0, 0))
+            task.subtask_count, task.closed_subtask_count = total_and_closed_subtask_count_per_parent_id.get(task.ids[0] if task.ids else False, (0, 0))
```

Comment:
Isn't `task` supposed to be singular? :monocle_face:

Reply:
It was due to bugs I had that I changed it but after trying now, It seems to be ok. Reversed to previous version :)